src/LanguageServer/Impl/Implementation/Server.cs

```diff
@@ -191,11 +192,11 @@ public sealed partial class Server : IDisposable {
             _disposableBag.ThrowIfDisposed();
             switch (@params.settings) {
                 case ServerSettings settings: {
-                        Settings = settings;
-                        _symbolHierarchyMaxSymbols = Settings.analysis.symbolsHierarchyMaxSymbols;
-                        _completionSource.Options = Settings.completion;
-                        break;
-                    }
+                    Settings = settings;
```

Comment:
Style nit, but these `{}` on switch case blocks should just be removed. They don't seem to add anything except noise in diffs because everyone's editors seem to disagree on the indention.

Reply:
sure. I can remove {}.
by the way, if only issue is "diff", then you can use "No whitespace" (?w=1) to remove those whitespace diff from PR.